### ISSUE TITLE
Error on multilingual push

### ISSUE THREAD

**Issue by AaronSadlerUK** [Jumoo/uSync.Complete.Issues#131]
When I try to push a node from development which does not exist on UAT, but only has a single language specificed I get the below error:

`"publishedContentHandler": Import Failed : "System.InvalidOperationException: Cannot save content with an empty name.`

For example this node has not been created in the Spanish varient
![image](https://user-images.githubusercontent.com/9937803/132213706-159a0a2c-ebc1-4e13-91c5-fa336929489c.png)

But has in the English varient, on the push window I am selecting just the English varient:
![image](https://user-images.githubusercontent.com/9937803/132213768-270b524f-8665-4552-82b1-b28577c41649.png)

But I get the above error

**Comment by KevinJump** [comment]
will take a look

**Comment by KevinJump** [comment]
think we have this - (its in uSync 😦 ) will push something when i've done some sanity checks.

**Comment by KevinJump** [comment]
Hi, 

we've pushed an update. 

the change is only to one file in uSync.ContentEdition so really you only need to update content edition. 

```
PM> update-package uSync.ContentEdition -version 8.10.1
```

but as this affects everything in the change, we've updated all the way up to uSync.Complete 8.11.1 (no changes except for the dependency on uSync 8.10.1). - so that will do a complete update. 

```
PM> update-package uSync.Complete -version 8.11.1
```

_(and this is one of the reasons we've merged content edition in v9 - it never makes sense to just update it)._
